pkgs/applications/networking/cluster/sonobuoy/default.nix

```diff
@@ -5,7 +5,7 @@ let rev = "5b97033257d0276c7b0d1b20412667a69d79261e";
 in
 buildGoModule rec {
   pname = "sonobuoy";
-  version = "0.56.13"; # Do not forget to update `rev` above
+  version = "0.56.14"; # Do not forget to update `rev` above
```

Comment:
Need to update the `rev` reference above.

Converting to draft.

Reply:
Resolved in https://github.com/NixOS/nixpkgs/commit/79b2a5bb3a6131eb4e20b47c1c4a58717b516fec, feel free to re-draft this if ofborg force-pushes this and drops the change again (I believe one of the maintainers is ["out until Jan 9"](https://github.com/saschagrunert)).

(Copy-paste from https://github.com/NixOS/nixpkgs/pull/199174#discussion_r1032567599: We really need a custom update script that can handle this :upside_down_face:)